NEWS.md

```diff
@@ -90,6 +90,7 @@ function calls. (#850, #851, @renkun-ken)
    + `expect_type_linter()` Require usage of `expect_type(x, t)` over `expect_equal(typeof(x), t)` and similar
    + `expect_s3_class_linter()` Require usage of `expect_s3_class(x, k)` over `expect_equal(class(x), k)` and similar
    + `expect_s4_class_linter()` Require usage of `expect_s4_class(x, k)` over `expect_true(methods::is(x, k))`
+   + `conjunct_expectation_linter()` Require usage of `expect_true(x)` over `expect_equal(x, TRUE)` and similar
```

Comment:
Description belongs to `expect_true_false_linter()`.

Reply:
Nice catch!